tools/autodeployment/server.js

```diff
@@ -53,35 +58,50 @@ function handleLog(req, res) {
   out.on('end', () => end('Build Complete.'));
 }
 
-const handleGitPush = createHandler({ path: '/', secret: SECRET });
+const handleGitPush = new Webhooks({ path: '/', secret: SECRET });
```

Comment:
The default hook path is already '/'.
https://github.com/octokit/webhooks.js#constructor

So we can pass just the secret.

path (String) | Only relevant for webhooks.middleware. Custom path to match requests against. Defaults to /.
-- | --